src/modules/accounting/prediction.py

```diff
@@ -0,0 +1,61 @@
+from typing import List, Iterable
+
+import web3.eth
+from web3 import Web3
+from web3.contract import ContractEvent
+from web3.types import Wei, EventData
+
+from modules.accounting.constants import REWARDS_HISTORICAL_PERIOD_IN_SECONDS, SECONDS_PER_SLOT
+from src.typings import BlockStamp
+
+
+class Prediction:
+    _eth_distributed: ContractEvent
+    _eth_distributed_address: web3.eth.Address = 'address'
```

Comment:
@F4ever 
I found an example in depositBot. 
https://github.com/lidofinance/depositor-bot/blob/main/scripts/utils/interfaces.py, that address and abi - is a GlobalConstant. This why it'll be change when constants come like in an examlpe